samza-core/src/main/java/org/apache/samza/standalone/PassthroughJobCoordinator.java

```diff
@@ -86,7 +86,7 @@ public void start() {
     try {
       jobModel = getJobModel();
       // TODO metrics registry has been null here for a while; is it safe?
-      MetadataResourceUtil metadataResourceUtil = new MetadataResourceUtil(jobModel, null, getClass().getClassLoader());
+      MetadataResourceUtil metadataResourceUtil = new MetadataResourceUtil(jobModel, null, getClass().getClassLoader(), config);
       metadataResourceUtil.createResources();
```

Comment:
Since TestRunner uses PassthroughJobCoordinator are the resources created (Changelog & Coordinator Stream) only when configs are present (job.coordinator.system & job.changelog.system with additional configs) ? Hence we need no changes on the TestRunner side?

Reply:
This is for checkpoint and changelog, not coordinator. The coordinator issue is a separate issue that does not affect version 1.2 and will be addressed in a separate PR.